src/Token/Stream.php

```diff
@@ -64,36 +64,44 @@ class PHP_Token_Stream implements ArrayAccess, Countable, SeekableIterator
 
     /**
      * @var array
+     *
+     * @psalm-var array{loc: int, cloc: int, ncloc: int}
      */
     protected $linesOfCode = ['loc' => 0, 'cloc' => 0, 'ncloc' => 0];
 
     /**
-     * @var array
+     * @var array|null
      */
     protected $classes;
 
     /**
-     * @var array
+     * @var array|null
      */
     protected $functions;
 
     /**
-     * @var array
+     * @var null|array
+     *
+     * @psalm-var null|array<string, list<string>>
      */
     protected $includes;
 
     /**
-     * @var array
+     * @var null|array
+     *
+     * @psalm-var null|array<string, array<string, mixed>>
      */
     protected $interfaces;
 
     /**
-     * @var array
+     * @var array|null
      */
     protected $traits;
 
     /**
      * @var array
+     *
+     * @psalm-var array<int, string>
```

Comment:
lineToFunctionMap is set at line 629 with int key and string value but is accessed on line 297 with string key. I think getFunctionForLine should take an int param.